plutus-core/untyped-plutus-core/test/Evaluation/Machines/NumberOfStepCounters.golden

```diff
@@ -0,0 +1 @@
+9
```

Comment:
We can also compute this number dynamically by looking at the enumeration of `StepKind`.  Would it be sensible to do that, or is this safer?

Reply:
Great idea, I've added that test alongside the other one. Thanks!